tensilelite/Tensile/Components/LocalRead.py

```diff
@@ -525,7 +525,7 @@ def __call__(self, writer, kernel, bufferIdx, iui, epsi, tP):
                                 incOffset = 0
                                 midIdx = numReadsPerUnroll // 2
                                 if rIdx >= midIdx:
-                                    if kernel["MatrixInstK"] > 32:
+                                    if  kernel["ProblemType"]["DataType"].is8bitFloat() and kernel["MatrixInstK"] > 32:
```

Comment:
These hard coding lines are stacking more and more.
In L524 you have HasMFMA_f8f6f4, and suddenly the condition in L528 becomes if ``is8bitfloat()`` and MIK > 32. Is MIK > 32 bounded to 8bit float?